app/scripts/controllers/transactions.js

```diff
@@ -201,6 +210,7 @@ module.exports = class TransactionController extends EventEmitter {
       nonceLock.releaseLock()
     } catch (err) {
       this.setTxStatusFailed(txId, {
```

Comment:
for those cases we should prolly write the error via `new Error(message)` instead of having this non standard format that contains the same info